### PR TITLE
Translate Ruby 2.5.0 Released (pt)

### PR DESCRIPTION
**Pull request by jcserracampos**

### CODE REVIEW

**Comment by jcserracampos** [comment]
@stomar Can you tell me where is missing whitespace?

File: pt/news/_posts/2017-12-25-ruby-2-5-0-released.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  você pode facilmente achar a linha causadora no final do seu _backtrace_.
+  Perceba que a ordem só é reversa quando o _backtrace_ é imprimido
+  diretamente no terminal.
+  [[Funcionalidade #8661]](https://bugs.ruby-lang.org/issues/8661) [experimental]
+
+## Melhorias de performance
+
+* Cerca de 5-10% de melhoria de performance ao remover todas as instruções `trace`
+  do _bytecode_ geral (sequências de instrução).
+  The `trace` instruction was added to support the `TracePoint`.
```
Comment (ZASMan) [inline pt/news/_posts/2017-12-25-ruby-2-5-0-released.md]: You forgot to translate this line

File: pt/news/_posts/2017-12-25-ruby-2-5-0-released.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
+---
+layout: news_post
+title: "Ruby 2.5.0 Released"
```
Comment (JuanitoFatas) [inline pt/news/_posts/2017-12-25-ruby-2-5-0-released.md]:
```suggestion
title: "Lançado Ruby 2.5.0"
```

😊

**Comment by JuanitoFatas** [comment]
Thanks! I fixed trailing spaces and merged as ab1d363.

**Comment by jcserracampos** [comment]
@JuanitoFatas I need to buy a xmas gift 👍

**Pull request closed without merging**